### DISCUSSION TITLE
Wasm size is increasing in .Net 9

### DISCUSSION THREAD

**Discussion by codemahek** [dotnet/aspnetcore#58914] [General]
![image](https://github.com/user-attachments/assets/63490e0d-700a-4b55-b006-e56bb53bf8f7)

Today I migrated one of my project from .Net 8 to .Net 9 and to my surprise wasm size of almost all of them has increased compared to .net 8 and especially MudBlazor was worst with 9.1 mb.
Is it because it’s not yet compiled with .net 9 or it’s a regression from asp.net

**Comment by gfoidl** [comment]
@codemahek I created an issue for this (https://github.com/dotnet/aspnetcore/issues/58920), so it becomes actionable.
Let's continue over there.
